Vagrantfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+    config.vm.box = "fedora/33-cloud-base"
+    config.vm.provider :virtualbox do |v|
+      v.memory = 2048
+      v.cpus = 2
+    end
+    config.vm.provider :libvirt do |v|
+      v.memory = 2048
+      v.cpus = 2
+    end
+
+    config.vm.synced_folder '.', '/vagrant', disabled: true
+
+    config.vm.provision "shell", inline: <<-SHELL
+      set -e -u -o pipefail
+      yum install -y git gcc docker
+      grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
+      service docker start
+    SHELL
+
+    config.vm.provision "shell", privileged: false, inline: <<-SHELL
+      curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+      echo "export PATH=$PATH:$HOME/.cargo/bin" >> ~/.bashrc
+
+      git clone https://github.com/containers/youki
```

Comment:
Why clone youki in rather than use the synced folder? I get that you might end up sharing build artifacts between platforms, but if you're using vagrant as your dev environment that shouldn't be too common of a problem. I think with this it just make it so you have to do all of your work in the VM rather than just the compilation steps.

Reply:
@tsturzl 
I believe that Vagrant is just for people who want to try youki out. That's why I wanted to leave out as some things as possible that might make that errors due to differences between platforms, such as sync.